src/openrct2/localisation/Formatting.cpp

```diff
@@ -0,0 +1,207 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2020 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#include "Formatting.h"
+
+#include "../config/Config.h"
+#include "../util/Util.h"
+#include "FormatCodes.h"
+#include "Language.h"
+#include "StringIds.h"
+
+#include <cstdint>
+
+namespace OpenRCT2
+{
+    char GetDigitSeperator()
+    {
+        return ',';
+    }
+
+    char GetDecimalSeperator()
+    {
+        return '.';
+    }
```

Comment:
Maybe these would be better of as `constexpr`

In any case, seems like "seperetor" has been mispelled.

Reply:
Fix spelling, constexpr not possible anymore.